src/java.base/share/classes/sun/security/ssl/X509TrustManagerImpl.java

```diff
@@ -51,6 +52,14 @@
 final class X509TrustManagerImpl extends X509ExtendedTrustManager
         implements X509TrustManager {
 
+    static {
+        try {
+            MethodHandles.lookup().ensureInitialized(AnchorCertificates.class);
```

Comment:
Why is this needed?  A comment would help.

Reply:
That's probably a good idea.  It’s caused by pinning due to the sun.security.util.AnchorCertificates’s class initializer, some of the http client tests are running into this. Once monitors are out of the way then class initializers, both executing, and waiting for, will be a priority.

Reply:
Added comment.